dashboards/app.py

```diff
@@ -0,0 +1,582 @@
+older_than = pd.Timestamp.now() - pd.Timedelta(days=1)
+flats_sold = flats_group_sold_by_day.df[(flats_group_sold_by_day.df['date_of_sale'] <=  older_than)] #per day
+flats_sorted_df = flats_sold.sort_values(by='last_seen_date')
+
+
+app = Dash(__name__)
+
+app.layout = html.Div(children=[
+
+    html.H1('Real estate prices and sales report in Gdynia'),
+
+    html.H3('Analysis of the prices of houses put up for sale'),
+    
+    html.P('Average daily price of houses for sale'), 
+    
+    dcc.Graph(
+        id='graph1',
+        figure=px.bar(avg_price_by_day , x="datetime", y="price")
+
+    ), 
+
+    html.P('Average daily price per meter of houses for sale'),
+
+    dcc.Graph(
+        id='graph2',
+        figure=px.bar(avg_price_per_meter_by_day, x="datetime", y="price_per_meter")
+    ),
+
+    html.P('Median daily price of houses for sale'),
+
+    dcc.Graph(
+        id='graph3',
+        figure=px.bar(median_price, x="datetime", y="price")
+    ),
+
+    html.P('Median daily price per meter of houses for sale'),
+
+    dcc.Graph(
+        id='graph4',
+        figure=px.bar(median_price_per_meter, x="datetime", y="price_per_meter")
+    ),
+
+    html.P('Average weekly price of houses for sale'),
+
+    dcc.Graph(
+        id='graph5',
+        figure=px.bar(avg_price_by_week, x="datetime", y="price")
+    ),
+
+    html.P('Average weekly price per meter of houses for sale'),
+
+    dcc.Graph(
+        id='graph6',
+        figure=px.bar(avg_price_per_meter_by_week, x="datetime", y="price_per_meter")
+    ),
+
+    html.P('Median weekly price of houses for sale'),
+
+    dcc.Graph(
+        id='graph7',
+        figure=px.bar(median_price_by_week , x="datetime", y="price")
+    ),
+
+    html.P('Median weekly price per meter of houses for sale'),
+
+    dcc.Graph(
+        id='graph8',
+        figure=px.bar(median_price_per_meter_by_week , x="datetime", y="price_per_meter")
+    ),
+
+
+    html.P('Average monthly price of houses for sale'),
+
+    dcc.Graph(
+        id='graph9',
+        figure=px.bar(avg_price_by_month, x="datetime", y="price")
+    ), 
+
+    html.P('Average monthly price per meter of houses for sale'),
+
+    dcc.Graph(
+        id='graph10',
+        figure=px.bar(avg_price_per_meter_by_month, x="datetime", y="price_per_meter")
+    ),   
+
+    html.P('Median monthly price of houses for sale'),
+
+    dcc.Graph(
+        id='graph11',
+        figure=px.bar(median_price_by_month , x="datetime", y="price")
+    ),   
+
+    html.P('Median monthly price per meter of houses for sale'),
+
+    dcc.Graph(
+        id='graph12',
+        figure=px.bar(median_price_per_meter_by_month , x="datetime", y="price_per_meter")
+    ),  
+
+    html.P('Number of houses for sale per day'),
+
+    dcc.Graph(
+        id='graph13',
+        figure=px.bar(number_of_houses_for_sale_per_day , x="datetime", y="scrapping_date")
```

Comment:
Czemu tutaj na osi y jest "scrapping_date"?

Reply:
Tutaj na osi Y powinnien być `count`

Reply:
wykres jest tworzony na podstawie kolumn scrapping_date i datetime , wyliczam to w zmiennej number_of_houses_per_day( linijka 16) , ale fakt jako legende trzeba to zmienić tyle ze musze sie doszukać jak .